style.css

```
@@ -0,0 +1,202 @@
+html {
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: normal;
+}
+
+body {
+  background-color: #eceef1;
+  display: grid;
+}
+
+a {
+  color: #0077c5;
+}
+
+nav {
+  display: grid;
+  justify-content: end;
+  text-align: right;
+  font-size: 12px;
+}
+
+input {
+  text-align: left;
+  height: 34px;
+  width: 95%;
+  min-width: 160px;
+  border-radius: 4px;
+  padding-left: 10px;
+  color: #000;
+  margin-top: -200px;
+  font-size: 16px;
+}
+
+input:focus {
+  border-color: blue;
+}
+
+footer {
+  text-align: center;
+  text-decoration: none;
+  margin-top: 21rem;
+  color: #505050;
+  font-size: 10px;
+}
+
+footer a {
+  text-align: center;
+  text-decoration: none;
+  color: #505050;
+}
+
+h1 {
+  color: #4c4c4c;
+  font-weight: 400;
+  line-height: 1.1;
+  font-size: 1.4rem;
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+.topSign {
+  grid-column: 3;
+  grid-row-start: 1;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+  background: #0077c5;
+  color: white;
+  margin-right: 2rem;
+  font-size: 1.125em;
+  font-weight: normal;
+  padding: 0 20px 0 20px;
+}
+
+.alText {
+  grid-column: 2;
+  grid-row: 1;
+  padding-right: 1rem;
+  margin-top: 2px;
+  margin-bottom: 1rem;
+  font-family: "AvenirNext", Avenir, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: normal;
+  color: #505050;
+}
+
+.fPass {
+  grid-column: 2;
+  grid-row: 1;
+  padding-right: 1rem;
+  padding-top: 1rem;
+}
+
+.links {
+  display: grid;
+  grid-template-columns: repeat(3, 1);
+  grid-template-rows: auto;
+  grid-gap: 2rem;
+  align-content: center;
+  margin-right: 32rem;
+  margin-left: 39.5%;
+  margin-bottom: 1.5rem;
+  max-width: 380px;
+  min-width: 380px;
+}
```

remove margin left and right here.

```suggestion
.links {
  display: grid;
  grid-template-columns: repeat(3, 1);
  grid-template-rows: auto;
  grid-gap: 2rem;
  align-content: center;
  margin-bottom: 1.5rem;
  max-width: 380px;
  min-width: 380px;
}
```